pkg/operation/seed/seed.go

```diff
@@ -845,11 +845,31 @@ func RunReconcileSeedFlow(
 		)
 		istioDeployers := []component.DeployWaiter{istioCRDs, istiod}
 
+		istioService := &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: *conf.SNI.Ingress.Namespace,
+				Name:      *conf.SNI.Ingress.ServiceName,
+			},
+		}
+
+		istioServiceKey := client.ObjectKeyFromObject(istioService)
+		err = seedClientSet.Client().Get(ctx, istioServiceKey, istioService)
+		if err != nil {
+			if !apierrors.IsNotFound(err) {
+				return err
+			}
+		}
+
+		currentAnnotations := map[string]string{}
+		if len(istioService.Annotations) > 0 {
+			currentAnnotations = istioService.Annotations
```

Comment:
For the time being, this is acceptable I think, however, eventually we want to refactor the istio deployment to use `ManagedResource`s as well, and then we don't need this custom coding any longer.